packages/tasks/src/model-libraries.ts

```diff
@@ -200,6 +200,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: true,
 		countDownloads: `path:"pytorch_model.bin"`,
 	},
+	SenseVoice: {
```

Comment:
Let's lowercase it to be consistent with other libraries. It requires to update https://huggingface.co/FunAudioLLM/SenseVoiceSmall/discussions/1 as well.

```suggestion
	sensevoice: {
```